### PR TITLE
fix for region change error

### PR DESCRIPTION
**Pull request by SmilingKat**
fixed the region change error by making a change to the way he
onRegionChange function is called

### CODE REVIEW

**Comment by lukekearney** [comment]
I can check it when I get home, but just wondering if this fixes the problem on the emulators?

File: TwMapView.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class TwMapView extends React.Component {
 				<MapView
 					style={styles.map}
 					initialRegion={this.state.region}
-					onRegionChange={this.onRegionChange}
+					onRegionChange={() => this.onRegionChange.bind(this)}
```
Comment (lukekearney) [inline TwMapView.js]: Just out of curiosity, does "this" refer to the MapView or the TwMapView component?

**Comment by lukekearney** [comment]
Ignore my previous comment about .lock files. Just read up on them and they're supposed to be checked in

**Comment by lukekearney** [comment]
Just going to merge this

**Comment by SmilingKat** [comment]
Thanks Luke, sorry I wasn't able to merge it during the last two days.

**Comment by lukekearney** [comment]
fixes #16

**Pull request merged**